integ-tests/src/main/java/com/google/pojo/TablesizeRow.java

```diff
@@ -0,0 +1,33 @@
+package com.google.pojo;
```

Comment:
Please add license header

Reply:
Oh, I forgot and missed that part. Will add to and double check other PRs.

Reply:
Done.